RedHat-UX/red-hat-design-system
# docs(audio player): update best practices

## What I did

1. Update format of audio player's "Best practices" section to use Do/Don't format


## Testing Instructions

1. Check ["Best practices" section](https://deploy-preview-2101--red-hat-design-system.netlify.app/elements/audio-player/guidelines/#best-practices) in the DP

## Notes to Reviewers
